synapse/lib/view.py

```diff
@@ -167,6 +167,16 @@ def getMergeRequest(self):
         if byts is not None:
             return s_msgpack.un(byts)
 
+    async def getMergingViews(self):
```

Comment:
Since the use case for this is a bit more transactional, we may want to also plumb a getMergingCount() API so that optic can just call it rather than needing to stream results back?

Reply:
Oh i had imagined we would just return the idens in a one-shot list?